checks.d/redisdb.py

```diff
@@ -233,9 +243,64 @@ def _check_db(self, instance, custom_tags=None):
                             slave_tags.append('slave_{0}:{1}'.format(slave_tag, info[key][slave_tag]))
                     slave_tags.append('slave_id:%s' % key.lstrip('slave'))
                     self.gauge('redis.replication.delay', delay, tags=slave_tags)
+    
+
+    def _check_slowlog(self, instance, custom_tags):
+        """Retrieve length and entries from Redis' SLOWLOG
+
+        This will parse through all entries of the SLOWLOG and select ones
+        within the time range between the last seen entries and now
+
+        """
+
+        max_slow_entries = "slowlog-max-len"
+        conn = self._get_conn(instance)
+
+        tags, tags_to_add = self._get_tags(custom_tags, instance)
+
+        if not instance.get(max_slow_entries):
+            max_slow_entries = int(conn.config_get(max_slow_entries)[max_slow_entries])
+            if max_slow_entries > DEFAULT_MAX_SLOW_ENTRIES:
+                self.warning("Redis {0} is higher than {1}. Defaulting to {1}."\
+                    "If you need a higher value, please set {0} in your check config"\
+                    .format(max_slow_entries, DEFAULT_MAX_SLOW_ENTRIES))
```

Comment:
This will result in having a string like

> Redis 256 is higher than 128. Defaulting to 128. If you need a higher value please set 256 in your check config

Which is not super understandable?